pages/osx/xsltproc.md

```diff
@@ -5,8 +5,8 @@
 
 - Transform an XML file with a specific XSLT stylesheet:
 
-`xsltproc --output {{output.html}} {{stylesheet.xslt}} {{xmlfile.xml}}`
+`xsltproc --output {{path/to/output_file.html}} {{path/to/stylesheet_file.xslt}} {{path/to/xmlfile_file.xml}}`
```

Comment:
```suggestion
`xsltproc --output {{path/to/output_file.html}} {{path/to/stylesheet_file.xslt}} {{path/to/file.xml}}`
```

Reply:
Nice suggestion. Furthermore it makes conversion result shorter as nothing appears before `file`. 👍🏻